data/birth-dates/17.txt

```diff
@@ -1 +1 @@
-Bertrand Russell: 18 May ????
```

Comment:
No, Bertrand Russel is older...

Reply:
Thanks for remind. I changed again(*￣︶￣)